config/config/common-config.yaml

```
@@ -349,6 +349,89 @@ prometheus:
         - name: elastisys
           pattern: ".*-elastisys-.*"
           limit: 80
+  diskAlerts:
+    storage:
+      predictLinear:
+        - hours: 24
+          percentage: 30
```

```suggestion
          FreeSpacePercentage: 30
```
maybe, just make it clear what this number are